crates/sncast/tests/docs_snippets/validation.rs

```diff
@@ -0,0 +1,75 @@
+use docs::validation::{
+    extract_matches_from_directory, extract_matches_from_file, get_parent_dir,
+    snippet_to_command_args,
+};
+use regex::Regex;
+use tempfile::tempdir;
+
+use crate::helpers::runner::runner;
+
+#[test]
+fn test_docs_snippets() {
```

Comment:
This is kind of duplicating the code we have in forge. Could this be made a reusable function or parts of it somehow?